src/content/release/breaking-changes/updated-material-3-progress-indicators.md

```diff
@@ -55,6 +55,21 @@ LinearProgressIndicator(
 ),
 ```
 
+To opt into the updated design spec for the `LinearProgressIndicator`, for the
+entire app, set the `ProgressIndicatorThemeData.year2023` flag to `false` in
+the `MaterialApp`:
```

Comment:
Consider adjusting the wording so "entire app" is earlier on in the sentence. Perhaps something like:

```suggestion
To update your entire app to use the updated `LinearProgressIndicator` design,
set the `ProgressIndicatorThemeData.year2023` property to `false` in your `MaterialApp`.
```